Dockerfile

```diff
@@ -73,7 +73,7 @@ WORKDIR /
 RUN mkdir -p /var/log/nginx /var/cache/nginx/ \
     && ln -s /dev/stdout /var/log/nginx/access.log \
     && ln -s /dev/sterr /var/log/nginx/error.log \
-    && useradd --create-home --user-group -u 999 --shell /bin/nologin nginx
+    && useradd --create-home --user-group -u 999 --shell /bin/false nginx

```

Comment:
is there a reason for this change?

Reply:
`/bin/nologin` binary is no longer available for Ubuntu 20.04. Another way to disable login for Nginx user is to use `/bin/false`. See this https://superuser.com/a/830694